.gitignore

```diff
@@ -1 +1,2 @@
-node_modules/
\ No newline at end of file
+node_modules/
+.vscode/
```

Comment:
Please add a new line here and everywhere where you have this kind of issue.

Reply:
Yeah, I was wondering why it is don't removing the folder

Reply:
@ricahcyuzuzo for that you have to untrack the folder.
read this to know how to do it `https://stackoverflow.com/questions/1274057/how-to-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore`